test/common/complexTransactions.js

```diff
@@ -1,32 +1,67 @@
 'use strict';
 
 var lisk = require('lisk-js');
+
+var node = require('../node');
 var http = require('./httpCommunication');
 
+function httpCallbackHelper (cb, err, res) {
+	if (err) {
+		return cb(err);
+	}
+	cb(null, res.body);
+}
+
+function getTransaction (transaction, cb) {
+	http.get('/api/transactions/get?id=' + transaction, httpCallbackHelper.bind(null, cb));
+}
+
+function getUnconfirmedTransaction (transaction, cb) {
+	http.get('/api/transactions/unconfirmed/get?id=' + transaction, httpCallbackHelper.bind(null, cb));
+}
+
+function getPendingMultisignature (transaction, cb) {
+	http.get('/api/multisignatures/pending?publicKey=' + transaction.senderPublicKey, httpCallbackHelper.bind(null, cb));
+}
+
 function sendTransaction (transaction, cb) {
-	http.post('/api/transactions', { transaction: transaction }, function (err, res) {
-		if (res.body.success) {
-			return cb(null, res.body);
-		}
-		return cb(res.body);
-	});
+	http.post('/api/transactions', {transaction: transaction}, httpCallbackHelper.bind(null, cb));
 }
 
 function sendSignature (signature, transaction, cb) {
-	http.post('/api/signatures', { signature: signature, transaction: transaction.id }, function (err, res) {
-		if (res.body.success) {
-			return cb(null, res.body);
-		}
-		return cb(res.body);
-	});
+	http.post('/api/signatures', {signature: {signature: signature, transaction: transaction.id}}, httpCallbackHelper.bind(null, cb));
```

Comment:
```
{ 
   signature: {
     signature: signature,
     transaction: transactionId
   }
}
```
Is this correct?

Reply:
It is.

Reply:
ok! looks weird to me :) the new api has it like this:

```
{
  "signatures": [
    {
      "transactionId": "222675625422353767",
      "signature": "2821d93a742c4edf5fd960efad41a4def7bf0fd0f7c09869aed524f6f52bf9c97a617095e2c712bd28b4279078a29509b339ac55187854006591aa759784c205"
    }
  ]
}
```

Reply:
Your guess is right. It is weird. But for now it works like that. We will adjust the code on the API refactoring.